lib/internal/url.js

```diff
@@ -621,7 +621,7 @@ function isURLThis(self) {
 }
 
 class URL {
-  constructor(input, base) {
+  constructor(input, base = undefined) {
```

Comment:
It looks like `undefined` is useless here, as **base** in anyway will be undefined in case it's `undefined` 😀 e.g.: buttery butter 🧈

Reply:
It's true that it doesn't change the behavior of the constructor, but the goal is to change the value of `URL.length` (from 2 to 1).